cloudify_cli/tests/test_utils.py

```diff
@@ -102,3 +105,92 @@ def test_dump_cosmo_working_dir_settings_create(self):
+
+        expected_dict = dict(my_key1="my_value1", my_key2="my_value2")
+
+        parsed_dict = utils.plain_string_to_dict(
+            "my_key1=my_value1;my_key2=my_value2")
+        self.assertEqual(parsed_dict, expected_dict)
+
+        parsed_dict = utils.plain_string_to_dict(
+            " my_key1 = my_value1 ;my_key2=my_value2; ")
+        self.assertEqual(parsed_dict, expected_dict)
+
+        parsed_dict = utils.plain_string_to_dict(
+            " my_key1 = my_value1 ;my_key2=my_value2; ")
+        self.assertEqual(parsed_dict, expected_dict)
+
+        expected_dict = dict(my_key1="")
+        parsed_dict = utils.plain_string_to_dict(" my_key1=")
+        self.assertEqual(parsed_dict, expected_dict)
+
+        parsed_dict = utils.plain_string_to_dict(" my_key1=;")
+        self.assertEqual(parsed_dict, expected_dict)
+
+        expected_dict = dict(my_key1="my_value1",
+                             my_key2="my_value2,my_other_value2")
+        parsed_dict = utils.plain_string_to_dict(
+            " my_key1 = my_value1 ;my_key2=my_value2,my_other_value2; ")
+        self.assertEqual(parsed_dict, expected_dict)
+
+        expected_err_msg = "Invalid input format: {0}, the expected " \
+                           "format is: key1=value1;key2=value2"
+
+        input_str = "my_key1"
+        self.assertRaisesRegexp(CloudifyCliError,
+                                expected_err_msg.format(input_str),
+                                utils.plain_string_to_dict, input_str)
+
+        input_str = "my_key1;"
+        self.assertRaisesRegexp(CloudifyCliError,
+                                expected_err_msg.format(input_str),
+                                utils.plain_string_to_dict, input_str)
+
+        input_str = "my_key1=my_value1;myvalue2;"
+        self.assertRaisesRegexp(CloudifyCliError,
+                                expected_err_msg.format(input_str),
+                                utils.plain_string_to_dict,
+                                input_str)
+
+        input_str = "my_key1=my_value1;my_key2=myvalue2;my_other_value2;"
+        self.assertRaisesRegexp(CloudifyCliError,
+                                expected_err_msg.format(input_str),
+                                utils.plain_string_to_dict,
+                                input_str)
+
+        input_str = "my_key1=my_value1;my_key2=myvalue2;my_other_value2;"
+        self.assertRaisesRegexp(CloudifyCliError,
+                                expected_err_msg.format(input_str),
+                                utils.plain_string_to_dict,
+                                input_str)
+
+        input_str = "my_key1:my_value1;my_key2:my_value2"
+        self.assertRaisesRegexp(CloudifyCliError,
+                                expected_err_msg.format(input_str),
+                                utils.plain_string_to_dict,
+                                input_str)
+
+    def test_inputs_to_dict(self):
```

Comment:
perhaps rename this test method to be more clear. 
It doesn't actually test `inputs_to_dict` it tests `inputs_to_dict` error handling on an invalid key=value type string input